polaris-react/src/components/IndexTable/IndexTable.scss

```diff
@@ -177,9 +176,13 @@ $loading-panel-height: 53px;
 }
 
 .TableHeading {
-  padding: var(--p-space-4);
+  background: var(--p-surface-subdued);
+  box-shadow: 0 1px var(--p-divider) inset;
```

Comment:
More of a design question, but is the box-shadow top border intended for the heading? The border acts as a divider for the other table items, but here it is adding a one-sided border to the table that seems inconsistent.

<img width="193" alt="12-05-s5r0v-1w62u" src="https://user-images.githubusercontent.com/11774595/184371652-1dc0cfa3-024b-4b83-9cba-6210ae5898a0.png">

Reply:
@samrose3 yeah, after a bit of consideration, especially with the `border-radius: inherit` being needed, I think the top dividing line as seen on the [Figma](https://www.figma.com/file/bZb9bEHNCEMiYjzO0fhK4J/Index-filters-and-search?node-id=682%3A183096) spec should exist outside of the IndexTable, and be the responsibility of the Filters component above to introduce a bottom border instead. Will remove this and reinstate the `border-radius: inherit`